recirq/quantum_chess/quantum_board.py

```diff
@@ -431,13 +442,17 @@ def _caching_supported(self, m: move.Move):
     def cache_results(self, cache_key: CacheKey):
         if cache_key in self.cache:
             return
+
+        self.reset_starting_states = False
+
         if cache_key.move_type == enums.MoveType.SPLIT_JUMP:
             helper_board = CirqBoard(
                 DEFAULT_CHESS_INIT_STATE,
                 self.sampler,
                 self.device,
                 self.error_mitigation,
                 self.noise_mitigation,
+                self.reset_starting_states,
```

Comment:
reset_starting_states=false?

Reply:
I tried doing it that way before, but it gave me  "SyntaxError: expression cannot contain assignment, perhaps you meant "=="." It works the way I have it, but happy to change it too, just not sure why it's giving me that error

Reply:
Maybe you wrote `self.reset_starting_states=False`

Reply:
I tried both, and with "reset_starting_states=False", there's no error, but the caching tests still fail

Reply:
Yeah, as I wrote in an earlier comment I don't expect the tests to pass without rebasing on #226.

Reply:
ah, makes sense. the PR is updated now

Reply:
mentioned this below but wanted to make sure you see - those few precache tests still fail after I committed the changes. Will look into it more

Reply:
Alright, should be good now